### PR TITLE
Update template with changes from Oflog dashboard

### PR DESCRIPTION
**Pull request by tom-clarke-dluhc**
## Pull request checklist

- [x] Add a descriptive message for this change to the PR
- [x] Run `black ./` locally
- [ ] Run `pylint <dashboard name>` locally and obtain a score of 10.00/10.00
- [x] Run `python -u -m pytest --headless tests` locally and there are no failures
- [x] Include screenshot for any visual changes

### PR Description:
- Change to LA codes rather than names and explain in comments
- Add an example visual test
- Add some accessibility changes
- Add performance profiling via snakeviz

image

### CODE REVIEW

**Review by Ellie-Brakoniecki — APPROVED** [review]
Great work, v useful to have this all in the template!

File: .github/workflows/check-before-merging.yml [hunk 1]
```diff
@@ -54,4 +55,70 @@ jobs:
           activate-environment: ${{ env.application-name }}
           environment-file: environment.yml
       - name: Execute tests
-        run: python -u -m pytest --headless tests 
+        run: python -u -m pytest --headless tests --ignore=tests/visual/
+
+  # this runs visual tests on Github, they have been more stable (less machine dependent) this way.
```
Comment (kateriley1) [inline .github/workflows/check-before-merging.yml:60]:
```suggestion
  # this runs visual tests as Github actions, they have been more stable (less machine dependent - GitHub Actions provides a consistent and isolated environment) this way.
```

File: .github/workflows/check-before-merging.yml [hunk 2]
```diff
@@ -1,8 +1,9 @@
 name: Automated checks
```
Comment (kateriley1) [inline .github/workflows/check-before-merging.yml:1]: Is it worth including workflows for cypress, attaching PR's to Trello & dependabot PR merge?

**Review by kateriley1 — APPROVED** [review]

File: .github/workflows/check-before-merging.yml (hunk 2 above)
Comment (tom-clarke-dluhc) [reply .github/workflows/check-before-merging.yml:1]: Definitely, thanks for suggesting. Will add these to the follow-up ticket

**Pull request merged**